src/lib/y2network/sysconfig/interface_file.rb

```diff
@@ -130,7 +130,7 @@ def initialize(name)
       # @return [IPAddr,nil] IP address or nil if it is not defined
       def ip_address
         str = fetch("IPADDR")
-        str.empty? ? nil : IPAddr.new(str)
+        str.nil? || str.empty? ? nil : IPAddr.new(str)
```

Comment:
we really have to adapt new rubocop to simply call just `str&.empty?`

Reply:
Good point.